src/coreclr/vm/methodtable.h

```diff
@@ -577,6 +578,9 @@ class MethodTable
 
     LPCWSTR GetPathForErrorMessages();
 
+    // This method boxes a value type using a given method table pointer
+    static FCDECL2(Object*, InternalBoxForMethodTable, MethodTable* pMethodTable, void* data);
```

Comment:
FCALL wrappers typically live in own files to keep the core implementation simpler. I would put this e.g. next to the existing `MethodTableNative::GetNumInstanceFieldBytes`.

Reply:
Done in 23b4a3ffa26e331db50ddce66353e82ce7913722.